meson.build

```diff
@@ -207,7 +207,7 @@ gen_hw_hdr = generator(
 # TODO: Considering moving these into hw/ip directories.
 TOPNAME='top_earlgrey'
 hw_ip_aes_reg_h = gen_hw_hdr.process('hw/ip/aes/data/aes.hjson')
-hw_ip_alert_handler_reg_h = gen_hw_hdr.process('hw/ip/alert_handler/data/alert_handler.hjson')
+hw_ip_alert_handler_reg_h = gen_hw_hdr.process('hw/' + TOPNAME + '/ip/alert_handler/data/autogen/alert_handler.hjson')
```

Comment:
Nit: Meson gets less grumpy about paths if you write `'hw' / TOPNAME / '...'`. IIRC this is a weird pythonism but Meson is very excited about those.

Reply:
The change here is consistent with the other files in the same block of code. I'll need to make some changes to that block rather soon, I'll remember to do the pathlib-style path cleanup as part of that as well.